### DISCUSSION TITLE
For each image on my page, livewire components are mounted

### DISCUSSION THREAD

**Discussion by ghost** [spatie/laravel-medialibrary#3656] [Q&A]
Hello everyone,

Here is the current configuration:

**php:8.2
laravel : 10.48.12
spatie-medialibary : 11.6.0
livewire: 3.4.1**

My problem is the following:
For each image present on my page, all livewire components visible on the same page are mounted for each iteration of the images. 
I've tested on pages where I don't have any images, and the components are mounted only once.

The major drawback is that these components make a lot of database requests and processing, and so I get serious slowdowns in my application.

Do you have the same problem?

**Comment by EkkoDig** [comment]
Same problem for me on a app
